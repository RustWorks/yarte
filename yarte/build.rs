#[path = "src/recompile.rs"]
mod recompile;

fn main() {
    recompile::when_changed();
}
