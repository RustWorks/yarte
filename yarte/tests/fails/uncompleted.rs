use yarte::Template;

#[derive(Template)]
#[template(src = "{{/o")]
struct Test;

fn main() {}
