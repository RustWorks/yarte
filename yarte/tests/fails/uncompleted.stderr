error: proc-macro derive panicked
 --> $DIR/uncompleted.rs:3:10
  |
3 | #[derive(Template)]
  |          ^^^^^^^^
  |
  = help: message: error
           --> $DIR/Test.hbs:1:3
            |
          1 | {{/o
            |   ^ problems parsing template source
            |
